src/main/java/org/testng/xml/ResultXMLParser.java

```diff
@@ -1,103 +0,0 @@
-package org.testng.xml;
-
-import org.testng.TestNGException;
-import org.testng.remote.strprotocol.GenericMessage;
-import org.testng.remote.strprotocol.IRemoteSuiteListener;
-import org.testng.remote.strprotocol.IRemoteTestListener;
-import org.testng.remote.strprotocol.SuiteMessage;
-import org.testng.remote.strprotocol.TestMessage;
-import org.testng.remote.strprotocol.TestResultMessage;
-import org.xml.sax.SAXException;
-
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
-
-/**
- * Parses testng-result.xml.
- *
- * @see ResultContentHandler
- *
- * @author Cedric Beust <cedric@beust.com>
- */
-public class ResultXMLParser extends XMLParser<Object> {
```

Comment:
I don't find it on https://github.com/testng-team/testng-remote/tree/rm_distributed_classes